LICENSES.txt

```diff
@@ -185,6 +190,7 @@ Apache-2.0
   netty-codec-stomp-4.1.89.Final.jar
   netty-codec-xml-4.1.89.Final.jar
   netty-common-4.1.93.Final.jar
+  netty-common-4.1.94.Final.jar
```

Comment:
Any hint on why do we get two versions in the LICENSES @gem-neo4j even if we are overwritting with the most recent one?

Reply:
The LICENSE file is a mix of core and common, so I believe common doesn't have 4.1.94 yet in its dependency tree

Reply:
Might be from test-utils actually, I see arrow is on 10.0.1 there, should that be updated?

Reply:
It's not that, I've tried overriding there as well. This should heal automatically anyway when the database updates to 4.1.94

Reply:
Yeah, should do :)